config.py

```diff
@@ -67,17 +67,24 @@ class Config(object):
 
     @staticmethod
     def init_app(app):
```

Comment:
I can't remember why we have this static method in `config.py` rather than say, `app/__init__.py`, or how it gets called? It'd be nice to keep `config.py` for well, config (not code). I don't know how difficult it is to move the stuff below somewhere else? For now, could we add a comment?

Reply:
Let me try moving it!

Reply:
Seems to work either way. Looking at the git log it isn't clear whether it was important for it to be in config.py or not.

Reply:
Yeah looking at the `git blame`, the method seems to have been there since the dawn of time 😸 I think it's just more noticeable the more complex it gets. Maybe leave for tech debt?